File: 250305/괄호 쌍 만들어주기 3/pair-parentheses-3.py
A = input()

# Please write your code here.
answer = 0

for i in range(len(A) - 1):
    if A[i] == ')':
        continue
    for j in range(i + 1, len(A)):
        if A[j] == ')':
            answer += 1

print(answer)

File: 260416/괄호 쌍 만들어주기 3/pair-parentheses-3.java
import java.io.*;

public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static void main(String[] args) throws IOException {
        // Please write your code here.
        String input = br.readLine();
        int answer = 0;

        for(int i = 0; i < input.length(); i++) {
            if(input.charAt(i) == '(') {
                for(int j = i; j < input.length(); j++) {
                    if(input.charAt(j) == ')') {
                        answer++;
                    }
                }
            }
        }
        System.out.println(answer);
    }
}